tools/gn

```diff
@@ -75,7 +75,7 @@ def to_gn_args(args):
 
   gn_args['clang_version'] = get_clang_version(args.target_toolchain)
   gn_args['api_version'] = args.api_version
-  gn_args['use_system_cxx'] = args.system_cxx
+  gn_args['no_system_cxx'] = args.no_system_cxx
```

Comment:
Please do not change the config name (I hate [double negation](https://dart.dev/effective-dart/design#prefer-the-positive-name-for-a-boolean-property-or-variable)). I was only referring to the command line option name in the above comment.

```suggestion
  gn_args['use_system_cxx'] = not args.no_system_cxx
```

Reply:
done